src/test/scala/circtTests/stage/ChiselStageSpec.scala

```diff
@@ -867,9 +867,11 @@ class ChiselStageSpec extends AnyFunSpec with Matchers with chiselTests.Utils {
 
     }
 
-    it("should emit specification FIRRTL (CHIRRTL)") {
+    it("should emit specification FIRRTL (CHIRRTL) with the correct FIRRTL spec version") {
 
-      ChiselStage.emitCHIRRTL(new ChiselStageSpec.Foo) should include("circuit Foo")
+      val text = ChiselStage.emitCHIRRTL(new ChiselStageSpec.Foo)
+      text should include("FIRRTL version 1.2.0")
+      text should include("circuit Foo")
```

Comment:
There may be a way to do this with:

```suggestion
      text should include allOf("FIRRTL version 1.2.0", "circuit Foo")
      text should include("circuit Foo")
```

Reply:
Annoyingly, I think `allOf` only works with thing like `contain`, not with `include`:
```
[error] /Users/jackk/work/chisel5/src/test/scala/circtTests/stage/ChiselStageSpec.scala:873:27: value allOf is not a member of ChiselStageSpec.this.ResultOf
IncludeWordForString                                                                                                                                        
[error]       text should include allOf ("FIRRTL version 1.2.0", "circuit Foo")                                                                             
[error]                           ^                                                                                                                         
```